### PR TITLE
Add a Rake task to download a prettified version of licenses.json

### PR DESCRIPTION
**Pull request by sschuberth**

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -1,5 +1,25 @@
...
+require 'net/https'
 require 'rake/testtask'
 
+task :download do
+  LICENSES_URI = URI.parse('https://spdx.org/licenses/licenses.json')
+
+  http = Net::HTTP.new(LICENSES_URI.host, LICENSES_URI.port)
+  if LICENSES_URI.scheme == 'https'
+    http.use_ssl = true
+    http.verify_mode = OpenSSL::SSL::VERIFY_NONE
```
Comment (domcleal) [inline Rakefile]: I don't think this is needed, verification should work.

File: Rakefile [hunk 2]
```diff
@@ -1,5 +1,25 @@
...
+  end
+
+  response = http.request_get(LICENSES_URI.path)
+  raise 'Downloading the licenses file failed.' unless response.is_a?(Net::HTTPOK)
```
Comment (domcleal) [inline Rakefile]: Nitpick, it'd be useful to include `response` in the error message.

File: Rakefile (hunk 1 above)
Comment (sschuberth) [inline Rakefile]: Actually it doesn't work, I've tried it:

    OpenSSL::SSL::SSLError: SSL_connect returned=1 errno=0 state=SSLv3 read server certificate B: certificate verify failed

File: Rakefile (hunk 2 above)
Comment (sschuberth) [inline Rakefile]: I'll include the response's message.

File: Rakefile (hunk 1 above)
Comment (domcleal) [inline Rakefile]: Oh, it did for me - perhaps a different in cert bundles or similar.  Not to worry.

**Comment by domcleal** [comment]
Thanks @sschuberth!

**Pull request merged**